microsoft/testsuites/cloud_hypervisor/ch_tests.py

```diff
@@ -98,8 +110,20 @@ def verify_cloud_hypervisor_live_migration_tests(
     ) -> None:
         hypervisor = self._get_hypervisor_param(node)
         ref = variables.get("cloudhypervisor_ref", "")
+        # below variable expects a comma separated list of full testnames
```

Comment:
This logic should be shared.

Reply:
You can use one function to return two variables, like
```python
a, b = get_lists(....)

def get_lists(...):
  return ["a"], ["b"]
```

Reply:
why return two lists? Isn't it generic to do it with one variable?

Reply:
Made changes to return two lists.